libraries/AR_Motors/AP_MotorsUGV.h

```diff
@@ -102,6 +102,8 @@ class AP_MotorsUGV {
     // true if vehicle has vectored thrust (i.e. boat with motor on steering servo)
     bool have_vectored_thrust() const { return is_positive(_vector_angle_max); }
 
+    bool is_omni() const { return _motors_num > 0; }
```

Comment:
let's put a comment above is_omni().. just so it's like the others.. maybe it's pretty obvious..